bigframes/eval.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 Google LLC
```

Comment:
Nit: This module name could get confusing if someone tries to call `bigframes.eval` instead of `bigframes.pandas.eval`.

I'm not sure how best to handle this, but maybe there's a sub-package we can use for this module like `core`?

Reply:
yeah, sounds good, moved to core.